examples/src/main/java/org/apache/ignite/examples/ml/tree/DecisionTreeClassificationTrainerExample.java

```diff
@@ -50,7 +50,7 @@ public static void main(String... args) throws InterruptedException {
                 // Create cache with training data.
                 CacheConfiguration<Integer, LabeledPoint> trainingSetCfg = new CacheConfiguration<>();
                 trainingSetCfg.setName("TRAINING_SET");
-                trainingSetCfg.setAffinity(new RendezvousAffinityFunction(false, 10));
+                trainingSetCfg.setAffinity(new RendezvousAffinityFunction(false, 1));
```

Comment:
Why did we change it on 1? It doesn't work with multiple partitions?

Reply:
I forgot to revert after debugging. Thanks